src/diff/index.js

```diff
@@ -228,7 +228,6 @@ export function diff(
  */
 export function commitRoot(commitQueue, root) {
 	if (options._commit) options._commit(root, commitQueue);
-
 	commitQueue.some(c => {
```

Comment:
The whitespace removal above here is probably unintentional :slightly_smiling_face:

Reply:
I thought that prettier did this 🤔

Reply:
Oh yes sorry I did not notice this let me fix this